src/App.tsx

```diff
@@ -15,37 +16,231 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+enum SortType {
+  Default = '',
+  Length = 'LENGTH',
+  Alphabet = 'ALPHABET',
+}
+
+interface FilterParams {
+  sortField: SortType | '';
+  isReversed: boolean;
+}
+
+function getPreperedGoods({ sortField, isReversed }: FilterParams): string[] {
+  const preperedGoods = [...goodsFromServer];
+
+  if (sortField) {
+    preperedGoods.sort((good1, good2) => {
+      switch (sortField) {
+        case SortType.Alphabet:
+          return good1.localeCompare(good2);
+
+        case SortType.Length:
+          return good1.length - good2.length;
+
+        default:
+          return 0;
```

Comment:
Would be great to move switch...case out of the sort callback to avoid calling sort method if the case is default